tests/Transports.AspNetCore.Tests/NewtonsoftJsonTests.cs

```diff
@@ -41,64 +41,57 @@ public async Task Decodes_BigInteger()
             var request = @"{""variables"":{""a"":1234567890123456789012345678901234567890}}";
             var ret = await Deserialize(request);
             var bi = BigInteger.Parse("1234567890123456789012345678901234567890");
-            ret.Single.Inputs["a"].ShouldBeOfType<BigInteger>().ShouldBe(bi);
+            ret.Single().Variables["a"].ShouldBeOfType<BigInteger>().ShouldBe(bi);
         }
 
         [Fact]
         public async Task Dates_Should_Parse_As_Text()
         {
             var ret = await Deserialize(@"{""variables"":{""date"":""2015-12-22T10:10:10+03:00""}}");
-            ret.Single.Inputs["date"].ShouldBeOfType<string>().ShouldBe("2015-12-22T10:10:10+03:00");
+            ret.Single().Variables["date"].ShouldBeOfType<string>().ShouldBe("2015-12-22T10:10:10+03:00");
         }
 
         [Fact]
         public async Task Extensions_Null_When_Not_Provided()
         {
             var ret = await Deserialize(@"{""variables"":{""date"":""2015-12-22T10:10:10+03:00""}}");
-            ret.Single.Extensions.ShouldBeNull();
-        }
-
-        [Fact]
-        public async Task Name_Matching_Not_Case_Sensitive()
-        {
-            var ret = await Deserialize(@"{""VARIABLES"":{""date"":""2015-12-22T10:10:10+03:00""}}");
-            ret.Single.Inputs["date"].ShouldBeOfType<string>().ShouldBe("2015-12-22T10:10:10+03:00");
+            ret.Single().Extensions.ShouldBeNull();
         }
```

Comment:
I removed this test.  Deserializing a `GraphQLRequest` object is always case-sensitive now, even when the underlying serializer is configured to be case-insensitive.

We could test what happens when deserializing an unrecognized object type, and then we would see that it is not case sensitive.  Or we could make this match the System.Text.Json serializer which is called "Name_Matching_Is_Case_Sensitive".

Reply:
>  Or we could make this match the System.Text.Json serializer which is called "Name_Matching_Is_Case_Sensitive".

OK. Catch exception and verify message.